docs/configuration/features/text_features.md

```diff
@@ -412,6 +404,7 @@ last vector of the sequence dimension).
 confidence_penalty: 0, robust_lambda: 0}`): is a dictionary containing a loss `type`. The only available loss `type` for
 text features is `softmax_cross_entropy`. For more details on losses and their options, see also
 [Category Output Features and Decoders](../category_features#category-output-features-and-decoders).
+- **`decoder`** (default: `{"type": "generator"}`): Decoder for the desired task. Options: `generator`, `tagger`.
```

Comment:
Same as above (unless this is already documented in the sequence features section). But also, there would be no harm to duplicate the explanations here so Ludwig users can reduce the number of clicks.

Reply:
Added link to decoder docs.